### DISCUSSION TITLE
Azura vs Shoutcast Server v2.6.1.777 double listening

### DISCUSSION THREAD

**Discussion by heyDaino** [AzuraCast/AzuraCast#5290] [Q&A]
Hello.

Why do I see double listeners? Also. not listening in SSL.
I think it's a SHOUTcast problem, maybe downgrading solves the problem.

What is the best version of Shoutcast? Unfortunately, I can't find any older versions online so that I can download (always if Azura allows it to downgrade Shoutcast)

Thank you!

UPDATE: the problem is only on Safari
UPDATEv2: downgraded to SHOUTcast Server v2.5.5.733 (maybe the best version) / no Shoutcast problem
UPDATEv3: no Azura problem, also stream hosted in Centova or WHM have same problem on Safari

Maybe is Safari problem
Sorry for these updates

<img width="878" alt="Schermata 2022-04-13 alle 10 31 53" src="https://user-images.githubusercontent.com/70600193/163134983-90ce1db4-c5a8-4726-b1a1-181dcd38fb42.png">

**Comment by BusterNeece** [comment]
@heyDaino I can't see any way that AzuraCast itself would be causing this...I think this must be a bug with SHOUTcast, in particular that version. I'm curious if installing a different version (if you can get your hands on one) resolves that issue.

**Reply by Vaalyn** [reply]
This is a bug with iOS Safari (I remember it from quite some time ago and which was previously fixed I think) that seems to have creeped back into Safari...

I can see this happening on Icecast as well as SHOUTcast so it's not related to the frontend used for a station.
It also doesn't matter if I access the streams via the nginx proxy or directly on the respective port.

This is a problem with the media playback in Safari itself. It is opening 2 connections to the stream instead of one.

When I'm back home I'll check if this is happening with iPadOS too and how it looks like with my outdate iPhone 6 with iOS 12 to verify if this is a regression in iOS Safari as mentioned above.
